src/core/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ if(NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 endif()
 
 # Allow CLOG to preprocess all the source files.
-add_clog_library(core.clog STATIC ${SOURCES})
+add_clog_library(core.clog SHARED ${SOURCES})
```

Comment:
Why shared? We got things working with a static LTTNg just fine. Also, we definitely don't want shared for Windows (when we get there).